src/CaravanTrait.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace StanfordCaravan;
+
+trait CaravanTrait {
+
+  public function tooldir(){
```

Comment:
This file needs linting. CC where you at?

Reply:
there it is. i'll address those this afternoon.